content/_data/changelogs/weekly.yml

```diff
@@ -93,6 +93,10 @@
       pull: 2683
     - type: rfe
       message: >
+        Searching in the <code>Build History</code> widget takes into account user preferences (case sensitivity by default).
```

Comment:
Why is everyone marking simple UI terms using `code`/`tt` etc? Makes no sense to me :cry: